README.md

```diff
@@ -1,3 +1,77 @@
 # @comet/dev-process-manager
 
+## Installation
+
+```console
+$ npm install
+$ npm run build
+```
+
 ## Usage
+
+Add `pm.config.js` file to the project root.
+This file defines all available apps, which should be started by dev-process-manager.
+
+### pm.config.js
+
+```javascript
+module.exports = {
+    apps: [
+        {
+             name: "api",
+             script: "npm --prefix api install && dotenv -- wait-on -l tcp:$POSTGRESQL_PORT && npm run --prefix api fixtures:dev && npm --prefix api run start:dev",
+        },
+        ...
+    ],
+};
+
+```
+
+Update dev script in package.json
+
+### package.json
+
+```json
+  ...
+  "dev": "dotenv -- node <path-to-dev-process-manager>/lib/index.js start pm.config.js",
```

Comment:
I only want to run `dev-pm start`

- config file should not be required (or optional)
- dotenv is application specific (not required in docs)
- dev-pm should install a bin file

Reply:
I think Alex said he would do the creation of the bin file as part of the publishing to npm
This will allow running it with `dev-pm start`
I can make the config file optional but then it could only run one script from one package.json
I removed dotenv from the expamle

Reply:
> I think Alex said he would do the creation of the bin file as part of the publishing to npm

ok. Do you have somewhere a todo-list for after-first-publish?

> I can make the config file optional but then it could only run one script from one package.json

You can keep the option and give it a default value. I'm unsure if that is a good idea, what do others say about that?

Reply:
> Do you have somewhere a todo-list for after-first-publish?
I just created a confluence page
> You can keep the option and give it a default value. I'm unsure if that is a good idea, what do others say about that?
I will check.